qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -153,7 +156,7 @@ def defaults(self, refresh=False):
 
         return self._defaults
 
-    def jobs(self, limit=10, skip=0, status=None, db_filter=None):
+    def jobs(self, limit=10, skip=0, status=None, db_filter=None, job_name=None):
```

Comment:
Can you reorder the arguments a bit, leaving `db_filter` as the last one? It has some penalty in regards to the signature of the function being backwards-compatible, but in this case I think we gain enough clarity by keeping the function parameters `(pagination, specific small filters, most generic filter)` (specially as the specific small filters are likely to be expanded in the future).